src/parastell.py

```diff
@@ -346,31 +351,27 @@ def export_dagmc(self, dagmc_export=dagmc_export_def):
                         (str, defaults to empty string).
                 }
         """
+        cubit_io.init_cubit()
+        
         self.logger.info(
             'Exporting DAGMC neutronics model...'
         )
 
-        global cubit_initialized
-
         export_dict = dagmc_export_def.copy()
         export_dict.update(dagmc_export)
```

Comment:
Any particular reason that the default DAGMC export dictionary is incorporated here and not in the `parastell()` function like the other default dictionaries?